docs/en/sql-reference/00-sql-reference/10-data-types/45-data-type-geospatial.md

```diff
@@ -0,0 +1,140 @@
+| Point              | A zero-dimensional geometric object, representing a specific location or coordinate point.                      | POINT(10 20)                                                                                      | POINT(-122.4194 37.7749) (San Francisco coordinates)                                               |
+| LineString         | A one-dimensional geometric object formed by a series of connected points, representing paths or line segments. | LINESTRING(10 20, 30 40, 50 60)                                                                   | LINESTRING(-122.4194 37.7749, -73.9352 40.7306) (San Francisco to New York)                        |
+| Polygon            | A two-dimensional geometric object with an outer ring and optional inner rings, representing areas.             | POLYGON((10 20, 30 40, 50 60, 10 20))                                                             | POLYGON((-122.5 37.7, -122.4 37.8, -122.3 37.7, -122.5 37.7)) (A region in San Francisco)          |
+| MultiPoint         | A collection of multiple zero-dimensional geometric objects.                                                    | MULTIPOINT((10 20), (30 40), (50 60))                                                             | MULTIPOINT((-122.4194 37.7749), (-73.9352 40.7306)) (Points in San Francisco and New York)         |
+| MultiLineString    | A collection of multiple LineString objects.                                                                    | MULTILINESTRING((10 20, 30 40), (50 60, 70 80))                                                   | MULTILINESTRING((-122.5 37.7, -122.4 37.8), (-122.3 37.7, -122.2 37.8)) (Multiple paths in a city) |
+| MultiPolygon       | A collection of multiple Polygon objects, representing multiple regions or areas.                               | MULTIPOLYGON(((10 20, 30 40, 50 60, 10 20)), ((15 25, 25 35, 35 45, 15 25)))                      | MULTIPOLYGON(((-122.5 37.7, -122.4 37.8, -122.3 37.7, -122.5 37.7))) (Multiple regions in a city)  |
+| GeometryCollection | A collection of different types of geometric objects, such as points, lines, and polygons.                      | GEOMETRYCOLLECTION(POINT(10 20), LINESTRING(10 20, 30 40), POLYGON((10 20, 30 40, 50 60, 10 20))) | GEOMETRYCOLLECTION(POINT(-122.4194 37.7749), LINESTRING(-122.5 37.7, -122.4 37.8))                 |
+
+## Supported Output Formats
+
+Databend supports three geospatial output formats—[WKT (Well-Known Text)](https://www.ogc.org/standards/sfa), [WKB (Well-Known Binary)](https://www.ogc.org/standards/sfa), and [GeoJSON](https://geojson.org/)—enabling compatibility with a wide range of geospatial applications and tools.
```

Comment:
better to use wikipedia link.
https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry
https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry#Well-known_binary